packages/image_picker/README.md

```diff
@@ -17,6 +17,17 @@ Add the following keys to your _Info.plist_ file, located in `<project root>/ios
 * `NSCameraUsageDescription` - describe why your app needs access to the camera. This is called _Privacy - Camera Usage Description_ in the visual editor.
 * `NSMicrophoneUsageDescription` - describe why your app needs access to the microphone, if you intend to record videos. This is called _Privacy - Microphone Usage Description_ in the visual editor.
 
+Or in text format add the key:
+
+``` xml
+<key>NSPhotoLibraryUsageDescription</key>
+<string>Used to demonstrate image picker plugin</string>
```

Comment:
I'm concerned that these examples would lead people to write poor explanations, since they are very vague. How about "Insert your explanation for using the photo library here", and similarly below.